CMakeLists.txt

```diff
@@ -129,27 +139,13 @@ endif()
 
 include_directories(${NMODL_PROJECT_SOURCE_DIR} ${NMODL_PROJECT_SOURCE_DIR}/src
                     ${PROJECT_BINARY_DIR}/src)
-include_directories(
-  SYSTEM ${NMODL_PROJECT_SOURCE_DIR}/ext ${NMODL_PROJECT_SOURCE_DIR}/ext/fmt/include
-  ${NMODL_PROJECT_SOURCE_DIR}/ext/spdlog/include)
-
-# =============================================================================
-# Include pybind11
-# =============================================================================
-message(STATUS "INCLUDING PYBIND11")
-add_subdirectory(ext/pybind11)
-
-# =============================================================================
-# Include fmt library
-# =============================================================================
-message(STATUS "INCLUDING FMT")
-add_subdirectory(${NMODL_PROJECT_SOURCE_DIR}/ext/fmt EXCLUDE_FROM_ALL)
-set_property(TARGET fmt PROPERTY POSITION_INDEPENDENT_CODE ON)
+include_directories(SYSTEM ${THIRD_PARTY_DIRECTORY} ${THIRD_PARTY_DIRECTORY}/fmt/include
```

Comment:
I think we can remove `${THIRD_PARTY_DIRECTORY}` there is nothing right there.

Reply:
IIRC, its needed because source include libraries as `<json/json.hpp>`